docs/development/best-practices.md

```diff
@@ -163,3 +164,14 @@ Use [JSDoc](https://jsdoc.app/index.html) to declare types and function prototyp
 - Omit constructors when defining Static classes
 - [No `#private` fields](https://google.github.io/styleguide/tsguide.html#private-fields). instead, use TypeScript's visibility annotations
 - Avoid underscore suffixes or prefixes, for example: `_prop`, use [whole words](https://google.github.io/styleguide/tsguide.html#properties-used-outside-of-class-lexical-scope) as suffix/prefix i.e. `internalProp`
+
+### regex
+
+- use [Named Capturing Groups](https://www.regular-expressions.info/named.html) when capturing groups (e.g `(?<groupName>CapturedGroup`)
```

Comment:
```suggestion
Use [Named Capturing Groups](https://www.regular-expressions.info/named.html) when capturing groups (e.g `(?<groupName>CapturedGroup`)
```

If there's one item, I don't think we should use a list. :wink: 

Are you sure about the example, it looks like you may be forgetting a `)` in the example string?

Reply:
i believe i wrote groups not group, but i added the word Multiple just to clear it out :D